src/app/components/Calendar/Calendar.styles.css

```diff
@@ -0,0 +1,25 @@
+.calendar {
+  background-color: var(--color-primary);
+  width: 100vw;
+  border-radius: var(--border-radius-card);
+  margin: 0;
+}
+
+.Calendar__weekDay {
```

Comment:
```suggestion
.calendar__weekDay {
```

Reply:
I want to overrule the default settings of the npm package I use for the calendar and need to take the classname from the package. It looks different than my classnames.